src/components/Ballot.test.tsx

```diff
@@ -1,15 +1,21 @@
 // Accessibility is not tested because ballot alone does not render html.
 // Each route component is tested elsewhere.
 
+import lodashMerge from 'lodash.merge'
 import React from 'react'
 import { fireEvent } from 'react-testing-library'
 
 import { render } from '../../test/testUtils'
 
+import electionDefaults from '../data/electionDefaults.json'
+import electionSample from '../data/electionSample.json'
+
 import Ballot from './Ballot'
 
 it(`can navigate all ballot pages`, () => {
-  const { container, getByText, getByTestId } = render(<Ballot />)
+  const { container, getByText, getByTestId } = render(<Ballot />, {
+    election: lodashMerge(electionDefaults, electionSample),
```

Comment:
It seems not quite right that the test file would have to know to merge the defaults with sample election. Feels like that should be within an abstraction of the core code, otherwise the tests strongly depend on this particular behavior.

Reply:
Agreed. I had done used this, then removed it for a reason that has no merit. 🏆

Reply:
On second thought… I remembered why which has merit… but there is a better way! Amending previous commit.